openedx/core/djangoapps/credit/models.py

```diff
@@ -19,6 +19,24 @@
 log = logging.getLogger(__name__)
 
 
+class CreditProvider(TimeStampedModel):
```

Comment:
Minor: There was no need to move this model. You could have setup your foreign key and many-to-many fields using the string path to the model name (`'credit.CreditProvider'`) instead of the reference to the class.